extensions/src/org/pentaho/platform/web/http/filters/PentahoPathDecodingFilter.java

```diff
@@ -40,8 +75,39 @@ public void destroy() {
   }
 
   private static final class EncodingAwareHttpServletRequestWrapper extends HttpServletRequestWrapper {
+
+    StringKeyStringValueIgnoreCaseMultivaluedMap parameterMap = null;
+    Map<String, String[]> parameters = null;
+
     private EncodingAwareHttpServletRequestWrapper( HttpServletRequest request ) {
       super( request );
+
+      if ( HttpMethod.GET.equalsIgnoreCase( request.getMethod() ) ) {
+        String query = getQueryString();
+        if ( StringUtils.isNotBlank( query ) ) {
+          try {
+            List<NameValuePair> pairs =
+                URLEncodedUtils.parse( new URI( "/?" + query ), request.getCharacterEncoding() );
+            parameterMap = new StringKeyStringValueIgnoreCaseMultivaluedMap();
```

Comment:
I believe that URL parameters are required to be case sensitive. The RFC states:
6.2.2.1. Case Normalization

When a URI uses components of the generic syntax, the component syntax equivalence rules always apply; namely, that the scheme and host are case-insensitive and therefore should be normalized to lowercase. For example, the URI is equivalent to http://www.example.com/.

The other generic syntax components are assumed to be case-sensitive unless specifically defined otherwise by the scheme (see Section 6.2.3).

So I don't believe that the parameterMap should use a StringKeyStringValueIgnoreCaseMultivaluedMap.